generate/template/astronomy.kt

```diff
@@ -7653,7 +7635,7 @@ private fun addSolarTerms(context: MoonContext) {$ASTRO_ADDSOL()}
 // Pluto state table
 
 $ASTRO_PLUTO_TABLE()
-private val plutoCache = HashMap<Int, List<BodyGravCalc>>()
+private val plutoCache = mutableMapOf<Int, List<BodyGravCalc>>()
```

Comment:
I am curious: what is the difference between `mutableMapOf` and `HashMap`? My only intention was to have O(1) random-access lookup of integer keys, like a sparse array.

Reply:
It shouldn't have any effective difference, HashMap is more like an implementation detail and mutableMapOf is an idiomatic language provided one, you can switch back if it feels wrong but it shouldn't be different.

Reply:
```
/**
 * Returns an empty new [MutableMap].
 *
 * The returned map preserves the entry iteration order.
 * @sample samples.collections.Maps.Instantiation.emptyMutableMap
 */
@SinceKotlin("1.1")
@kotlin.internal.InlineOnly
public inline fun <K, V> mutableMapOf(): MutableMap<K, V> = LinkedHashMap()
```

HashMap vs LinkedHashMap: https://www.javatpoint.com/linkedhashmap-vs-hashmap-in-java HashMap can be considered more so perhaps can be reverted.